pt/2/7-storage.md

```diff
@@ -118,15 +118,15 @@ Por enquanto, é o suficiente para entender que há casos onde você precisará
 
 # Vamos testar
 
-Esta na hora de dar aos nossos zumbis a habilidade de alimentar e multiplicar!
+Esta na hora de dar aos nossos zumbis a habilidade de alimentar-se e multiplicar-se!
 
 Quando um zumbi se alimenta de outra forma de vida, o seu DNA irá combinar com o DNA da outra forma de vida criando um novo zumbi.
 
 1. Crie uma função chamada `feedAndMultiply`. Que terá dois parâmetros: `_zombieId` (um `uint`) e `_targetDna` (também um `uint`). Esta função deverá ser `public` (pública).
 
 2. Nós não queremos que qualquer um alimente nossos zumbis! Então primeiro, vamos ter certeza que temos posse deste zumbi. Adicione uma declaração `require` para ter certeza que `msg.sender` é igual ao dono do zumbi (similar a que fizemos antes na função `createRandomZombie`).
 
- > Nota: Novamente, devido ao nosso checador-de-resposta ser simples, ele espera que o `msg.sender` venha primeiro e irá marcar errado se você trocar a ordem. Mas normalmente quando você estiver codando, você pode usar da forma que quiser - ambas são corretas.
+ > Nota: Novamente, devido ao nosso verificador de respostas ser simples, ele espera que o `msg.sender` venha primeiro e irá marcar errado se você trocar a ordem. Mas normalmente quando você estiver escrevendo seus códigos, você pode usar da forma que quiser - ambas são corretas.
```

Comment:
I think it should have an `article 'os'` on `escrevendo seus códigos` to be `escrevendo os seus códigos`, according to https://ciberduvidas.iscte-iul.pt/consultorio/perguntas/o-uso-do-determinante-e-pronome-seu-com-sujeito-plural/30703